index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/main.css">
+  <title>Art Blog ~ Home Page</title>
+</head>
+
+<body>
+  <div class="container">
+
+    <!-- Top Navigation -->
+    <section class="navigation">
+      <nav class="navigation__top-nav">
+        <ul>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-menu">
+              <use xlink:href="assets/Icons/sprite.svg#icon-facebook"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <svg class="nav-item__icon icon-cross">
+              <use xlink:href="assets/Icons/sprite.svg#icon-twitter"></use>
+            </svg>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">English</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">My Page</a>
+          </li>
+          <li class="nav-item">
+            <a href="" class="nav-link">Logout</a>
+          </li>
+        </ul>
+      </nav>
+    </section>
+
+    <!-- Header Section -->
+    <header class="header">
+
+      <!-- Headre Nav -->
+      <section class="header-nav">
+        <div class="header-nav-container">
+          <div class="logo-box"><span>Art</span>~Blog</div>
+          <nav class="header-nav-container__nav-bar">
+            <div class="header-nav-container__nav-bar--icon" id="hamberger">
+              <svg class="header-nav-container__nav-bar--icon" id="icon-menu">
+                <use xlink:href="assets/Icons/sprite.svg#icon-menu"></use>
```

Comment:
- [ ] The hamburger should be on the left hand side of the page as it is in the original design. Kindly make it match the original design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157758337-4ccf22f7-16c6-41a3-bedf-1b3f152b9ffd.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157758176-48dd2538-cae5-436e-b81e-61418986c96e.png)